packages/transformers/json/src/JSONTransformer.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+
+export default new Transformer({
+  async transform({asset}) {
+    asset.type = 'js';
+    asset.setCode(`module.exports = ${await asset.getCode()};`);
+    return [asset];
```

Comment:
Ideally we’d also support parsing and re-stringifying json5 as well like parcel 1, but that could be added later.

Reply:
Should json5 be part of the default JSON transformer? Would be convenient, but it would be different than, for example, how Node loads json (which would throw if it encountered anything nonstandard).

If so, it should be a pretty straightforward here :)

Reply:
JSON5 is pretty widely adopted by various tools at this point...

Reply:
Sorry, what I meant was "should we load all json files through json5?" A lot of tools these days at the very least ignore comments, etc. in JSON despite them being named `.json`. This would be different behavior from Parcel 1, though.

Reply:
Yeah that's what json5 is right? Allowing comments etc. In Parcel 1 we do allow them in `.json` files